core/storage/suggestion/gae_models.py

```diff
@@ -470,7 +470,8 @@ def get_in_review_suggestions_in_score_categories(
 
     @classmethod
     def get_in_review_translation_suggestions(
-        cls, user_id: str, language_codes: List[str]
+        cls, user_id: str, language_codes: List[str],
+            exp_ids: List[str]
     ) -> Sequence[GeneralSuggestionModel]:
         """Gets all translation suggestions which are in review.
```

Comment:
This now seems wrong perhaps (or may need a bit more elaboration)? (Admittedly, it was vague before too.)

Reply:
Added more elaboration here.

Reply:
But that's still not true -- this method is only getting them for particular languages / exps / topic names, rather than all in-review translation suggestions in the datastore.

Reply:
[Same response as for one of the below comments, just copying here for reference].  
I have been slightly confused since in some of the case cases, while explaining the input arg. in detail, it essentially turns out to be a repetition of the description. That is why here, I let the description be crisp and explained in detail below the description (as part of the input arg).

But as you described, the description can be a more conceptual understanding, and input arg can be more detailed.

Reply:
I think the main thing is that it's got to be correct. It's fine for it to be long, and it's fine for there to be repetition, as long as each description is clear and correct.

Brevity isn't something to optimize for at the expense of correctness (at least in this codebase). I would suggest using as many words as are necessary to describe each thing correctly -- and no more :P

Reply:
(Note that this philosophy also helps keep the codebase accessible to newer contributors; tight-knit teams with a strong shared understanding and a not-too-big codebase might make different choices.)